RevenueCatUI/Templates/V2/ViewHelpers/Fill.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Fill.swift
+//
+//  Created by Mark Villacampa on 24/1/25.
+
+import RevenueCat
+import SwiftUI
+
+#if PAYWALL_COMPONENTS
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+extension Shape {
+
+    func fillColorScheme(
```

Comment:
We already have helpers to apply a foreground and background color scheme but I needed to `fill` a shape. 

At some point we need to refactor and unify the logic of all these methods.